src/core/public/saved_objects/saved_objects_client.ts

```diff
@@ -348,7 +384,25 @@ export class SavedObjectsClient {
       workspaces: 'workspaces',
     };
 
-    const renamedQuery = renameKeys<SavedObjectsFindOptions, any>(renameMap, options);
+    const currentWorkspaceId = this.currentWorkspaceId;
+    let finalWorkspaces;
+    if (options.hasOwnProperty('workspaces')) {
+      finalWorkspaces = options.workspaces;
+    } else if (typeof currentWorkspaceId === 'string') {
+      finalWorkspaces = Array.from(new Set([currentWorkspaceId]));
+    }
+
+    const renamedQuery = renameKeys<Omit<SavedObjectsFindOptions, 'ACLSearchParams'>, any>(
```

Comment:
seems `ACLSearchParams` doesn't exist at the moment

Reply:
You are right, remove that.